backends-clickhouse/src/main/scala/org/apache/gluten/backendsapi/clickhouse/CHBackend.scala

```diff
@@ -130,6 +130,12 @@ object CHBackendSettings extends BackendSettingsApi with Logging {
 
   val GLUTEN_AQE_PROPAGATEEMPTY: String = CHConf.prefixOf("aqe.propagate.empty.relation")
 
+  val GLUTEN_ALLOW_READ_JSON: String =
+    GlutenConfig.GLUTEN_CONFIG_PREFIX + CHBackend.BACKEND_NAME +
```

Comment:
use CHConf.runtimeSettings instead of `GlutenConfig.GLUTEN_CONFIG_PREFIX + CHBackend.BACKEND_NAME+".runtime_settings"`